pkgs/tools/inputmethods/uim/default.nix

```diff
@@ -53,6 +54,10 @@ assert withNetworking -> curl != null && openssl != null;
 assert withFFI -> libffi != null;
 assert withMisc -> libeb != null;
 
+let
+  stdenv = if args.stdenv.cc.isGNU then args.gcc13Stdenv else args.stdenv;
```

Comment:
You have verified this issue isn't observed on Clang?

Reply:
This was caused by the gcc update, so only gcc stdenvs should be modified. This changes nothing for clang stdenvs and it shouldn't have broken by the recent gcc14 bump.

We wouldn't want to force stdenvs that are clang by default to gcc, so this is the way.